libs/domain/content/package.json

```diff
@@ -23,6 +23,8 @@
     "@spryker-oryx/utilities": "1.3.1"
   },
   "peerDependencies": {
+    "marked": "^5.1.0",
+    "@types/marked": "^5.0.0",
```

Comment:
can we bump to latest version. `11.0.0`. Also, let's use major versions only as we're using caret, unless the package does not respect semver.